### PR TITLE
add Devise

### PR DESCRIPTION
**Pull request by hereismyusername**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -8,7 +8,13 @@
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
-  <body>
-    <%= yield %>
+  <body
```
Comment (cldambrosio) [inline app/views/layouts/application.html.erb]: missing closing tag?

Comment (hereismyusername) [reply app/views/layouts/application.html.erb]: Yes - thanks!

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -9,6 +9,12 @@
...
 
   <body>
-    <%= yield %>
+    <% if notice.present? %>
+      <p class="alert alert-info"><%= notice %></p>
+      <% end %>
+      <% if alert.present? %>
+      <p class="alert alert-danger"><%= alert %></p>
+      <% end %>
+      <%= yield %>
```
Comment (rosalinekarr) [inline app/views/layouts/application.html.erb]: You'll probably want to fix the tabbing here. The code is fine, but the whitespace should look more like this:

```
 +    <% if notice.present? %>
 +      <p class="alert alert-info"><%= notice %></p>
 +    <% end %>
 +    <% if alert.present? %>
 +      <p class="alert alert-danger"><%= alert %></p>
 +    <% end %>
 +    <%= yield %>
```

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
+end
```
Comment (rosalinekarr) [inline app/models/user.rb]: This is a little beyond the scope of this task, but that's not a problem. You just did all @RajSannidhi's work for him!

**Pull request merged**